### ISSUE TITLE
How to add bookmark on the explorer using hotkey?

### ISSUE THREAD

**Issue by ilovefrontendph** [UrosVuj/Explorer-Bookmark#6]

**Comment by dimateos** [comment]
Yeah woul be great to be able to add an opened file to the list with a command/hotkey, instead of right-clicking on the explorer view. Maybe being able to add multiple selected files at one helps with the issue of having to only use the tree view tho!

cheers thanks for the extension!

**Comment by dimateos** [comment]
To clarify a bit and draw some attention. I would add 2 commands that would do:
* Add open file to bookmarks
* Add parent folder of open file to bookmarks

For completeness, but would not be that useful:
* Delete open file to bookmarks
* Delete parent folder of open file to bookmarks

And in case of renaming feature being added:
* Rename open file to bookmarks
* Rename parent folder of open file to bookmarks

This would be convinient, but IMO what I really need is the ADD command.
Deleting/renaming could be done from the UI in the bookmarks panel.
